UnityProject/Assets/Scripts/Player/ExtinguishByHand.cs

```diff
@@ -0,0 +1,29 @@
+using UnityEngine;
+
+/// <summary>
+/// Allows an object to be patted when on fire.
+/// </summary>
+public class ExtinguishByHand : MonoBehaviour, IClientInteractable<PositionalHandApply>
+{
+	public bool Interact(PositionalHandApply interaction)
```

Comment:
This should be doable without needing to add a player network action cmd, by using ICheckedInteractable instead of IClientInteractable. You don't have to refactor it to do this, but I just wanted to make sure you're aware of that, it's usually preferable to adding to PNA. Maybe it doesn't work due to precedence issues (which are planned to be fixed but will probably be a bit before that happens)? Precedence shouldn't be a problem as long as you put this below Meleeable.

Reply:
Ah, well I'll try to make it a ICheckedInteractable.

I was hoping to implement ExtinguishByHand by taking Huggable as an example. But it seems like that wasn't the way to go.